### ISSUE TITLE
[question] is this supposed to be possible?

### ISSUE THREAD

**Issue by macks2008** [Talia-12/Hexal#86]
You might've already seen the discussion we had in #Hermes-other-gambit after I noticed this, but just in case you didn't:
you know how you can combine a true name and a cleric impetus to be able to cast as someone else? It occurred to me you might be able to modify someone's Everbook the same way, which I realized may or may not be intended since normally true name "hacks" only affect the current save/server.
Of course, I said "might", since I'm not sure if it is actually possible. It's a bit difficult to test something like this since it explicitly requires someone trusting you with their true name (I've only been able to obtain one so far and on a server where I don't have the resources for circle magic yet)

**Comment by Talia-12** [comment]
that's not *great*, not super pleased about it, but I can't think of any ways to fix it that are satisfying; I don't want to make the everbook patterns staff-cast only, and that's the only thing I can really think of that'd make this not a problem.

**Comment by macks2008** [comment]
Yeah I couldn't think of much of anything either. It would be nice if you could ask Hexcasting's API "hey is this pattern DEFINITELY being cast by the focus's owner, or did someone else, not the focus's owner, link the focus to a circle". Failing something like that, maybe you could at least make it so that the user can control, somehow (preferably without requiring a pattern itself, or at least without allowing the adversary to disable this by shadowing a pattern with a macro), whether a given world can modify their Everbook. 
of course, if someone truly malicious has your True Name, you might want to reconsider playing in that server in the first place, and I don't think any of this works if you are not online, right? (Since even if they had a Figment or something, there would be no client for it to get the Everbook from). The most I would do with this knowledge is steal someone's entries

**Comment by Talia-12** [comment]
yeah there's definitely no way to affect your Everbook if you aren't online. A per-world config doesn't really feel like a useful way to fix the issue? If someone thinks to actually make use of it they'll probably also think of backing the everbook up somewhere, which doesn't rely on my ability to patch against adverse behaviour.
